index.html

```diff
@@ -201,7 +201,7 @@ <h5 class="intro2">LET'S CONNECT</h5>
         <a href="#"><img src="images/fingers.png" alt="fingers"></a>
     </li>
  </ul>
-
+ 
```

Comment:
- [ ] Great job. please ensure that the hamburger does not appear on desktop version. the menu links should show up instead like in the original design
- [ ] kindly centralize your headline like in the figma design

yours
![Screenshot (663)_LI](https://user-images.githubusercontent.com/65298048/206854802-4668c547-b777-4aee-ba6e-c685dcdc2787.jpg)
figma design
![Screenshot (664)](https://user-images.githubusercontent.com/65298048/206854824-7c130cd4-0f3a-4670-a58f-1e361f45f99c.png)